src/Modules/SimplCommerce.Module.Localization/EFStringLocalizer.cs

```diff
@@ -61,17 +61,17 @@ private string GetString(string name)
             var resources = LoadResources(culture);
             var value = resources.SingleOrDefault(r => r.Key == name)?.Value;
 
-            //if (value == null)
-            //{
-            //    AutoRegisterNewString(name, culture);
-            //}
+            if (value == null)
+            {
+                AutoRegisterNewString(name, culture);
+            }
 
             return value;
         }
 
         private void AutoRegisterNewString(string name, string culture)
         {
-            if (culture == "en-US") //GlobalConfiguration.DefaultCulture
+            if (culture != "en-US") //GlobalConfiguration.DefaultCulture
```

Comment:
Could please let me know why you negate the condition?

Reply:
I need for all languages, which are not en-US the possibility to choose it on the translations page and to enter the respective translations.
In my personal case I'm working with de-DE and en-GB, so I can enter the translations for those two languages ...

Reply:
My initial question regarding this change was:
Can everybody agree that the standard texts in the application can be considered en-US (e.g. @localizer[“Cancel”]), which means that all other languages can be considered a translation?

Reply:
> Can everybody agree that the standard texts in the application can be considered en-US (e.g. @localizer[“Cancel”]), which means that all other languages can be considered a translation?

Agree, that's the default